Library/Homebrew/rubocops/options_cop.rb

```diff
@@ -43,5 +43,15 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
         end
       end
     end
+
+    module NewFormulaAudit
+      class Options < FormulaCop
+        MSG = "New Formula should not use `deprecated_option`".freeze
```

Comment:
Newline after `MSG` (and in general can you go and fix that up for any other occurrences in this or other cops?)